packages/auth-helpers/src/lib/models.ts

```diff
@@ -173,4 +173,5 @@ export interface ILitResource {
 export type LitResourceAbilityRequest = {
   resource: ILitResource;
   ability: LitAbility;
+  data?: any;
```

Comment:
Aren't there only 2 fields? What is `data` used for?

Reply:
I don't think is being used, but `data` can be used for the `proof` field if we decide to use that in the nodes to proof something. I can remove this.

Reply:
Let's keep it as it may be used but not being used currently. This can be used to check the proof on the nodes